### PR TITLE
docs: fix wordos

### PR DESCRIPTION
**Pull request by olizilla**
minor typo/wordo fixes

License: MIT
Signed-off-by: Oli Evans <oli@tableflip.io>

### CODE REVIEW

File: schemas/authoring-guide.md [hunk 1]
```diff
@@ -41,7 +41,7 @@ The DSL treats newlines as significant, they are used to break up records (`type
...
 
-Whitespace characters (tab and space) and also folded in to a single space during parsing, so may be used for formatting and documentation purposes where appropriate. Most tokens that don't need to be proceeded by a newline should be separated by at least one newline character. There are other tokens that don't strictly require a newline (e.g. `{String:Int}` for Map definitions where 5 tokens may be conjoined, but also may be separated, `{ String : Int }`). Indenting is not strictly required for record component descriptors but are typical as they can be used to express intent.
+Whitespace characters (tab and space) are also folded in to a single space during parsing, so may be used for formatting and documentation purposes where appropriate. Most tokens that don't need to be proceeded by a newline should be separated by at least one newline character. There are other tokens that don't strictly require a newline (e.g. `{String:Int}` for Map definitions where 5 tokens may be conjoined, but also may be separated, `{ String : Int }`). Indenting is not strictly required for record component descriptors but are typical as they can be used to express intent.
```
Comment (olizilla) [inline schemas/authoring-guide.md:44]: Whitespace characters (tab and space) **are** also folded

File: schemas/introduction.md [hunk 2]
```diff
@@ -6,11 +6,11 @@ IPLD concerns itself with the data layer of the distributed web. Its scope begin
...
+IPLD’s Data Model is useful for building an abstraction at the data layer. It pushes concerns about encoding and decoding to its own dedicated domain where such concerns only have to deal with a limited set of data types and how to convert to and from those data types. But it also makes room for the development of an ecosystem of libraries and tools that are able to build on top of this model and can therefore operate according to shared assumptions about how data is presented. Such an ecosystem should empower significantly more sharing of algorithms, tools, techniques and code in the distributed web community, rather than siloing these assets in codebases that are concerned entirely with use-case-specific stacks.
```
Comment (olizilla) [inline schemas/introduction.md:9]: codebases that are concerned entirely **with** use-case-specific stacks.

**Comment by olizilla** [comment]
github diff is doing a terrible job of highlighting what is acutally changed here, so i've commented inline

**Review by rvagg — APPROVED** [review]

**Comment by rvagg** [comment]
nice to see someone else actually reading them! thanks for the fixes Oli

**Pull request merged**